backend/tests/main.py

```diff
@@ -483,3 +485,109 @@ def test_auth_login_google():
         "/auth/login/google", json=post_body, cookies=response.cookies
     )
     assert response.status_code == 200
+
+
+@vcr.use_cassette(record_mode="once")
+def test_fakewallet(client):
+    from app import config
+    if config.settings.stripe_public_key:
+        pytest.skip("Stripe is configured")
+    # Complete a login through Github
+    response = client.get("/auth/login/github")
+    assert response.status_code == 200
+    out = response.json()
+    assert out["state"] == "ok"
+    state = dict(parse.parse_qsl(parse.urlparse(out["redirect"]).query))["state"]
+    print(state)
+    post_body = {"code": "04f6dff87ead3551df1d", "state": state}
+    response = client.post(
+        "/auth/login/github", json=post_body, cookies=response.cookies
+    )
+    assert response.status_code == 200
+
+    # Test the login was success through `auth/userinfo`
+    response = client.get("/auth/userinfo")
+    assert response.status_code == 200
+    out = response.json()
```

Comment:
Did you check if there's anything in here, that you shouldn't post online/change?